scripts/cdist

```diff
@@ -27,7 +27,7 @@ def inspect_ssh_mux_opts(control_path_dir="~/.ssh/"):
     import os
 
     control_path = os.path.join(control_path_dir,
-            "cdist.socket.master-%l-%r@%h:%p")
+            "s-%r@%h:%p")
```

Comment:
Isn't the socket LOCAL to each run anyway? I think we don't even need the variable ssh part in it, because the socket is always within our tempdir for the run and could just use "ssh-control-path" as a static name.

Or am I wrong here?